docs/tutorials/intro.md

```diff
@@ -0,0 +1,177 @@
+   ![View Shapefile Layer](../assets/images/viewShapefileLayer.png)
+
+:::{admonition} Why don't we add a local shapefile?
+:class: tip
+One of the main strength of JupyterGIS is its collaborative capabilities. To be able to collaborate, it is much easier to work with data shared and available online.
+:::
+
+```{exercise-start}
+:label: add-vector-layer
+```
+
+- Add a new Shapefile Layer to your map:
+
+```
+https://docs.geoserver.org/stable/en/user/_downloads/30e405b790e068c43354367cb08e71bc/nyc_roads.zip
+```
+
+- Zoom over New-York (USA) and check if you can see the newly added layer.
+- Let's customize this new layer by changing the color. In the GIS Layer/Browser Panel, select the top layer (corresponds to the last layer you added) and right click to **Edit Symbology**. Then change the **Stroke Color** to a color of your choice. You can also change the Stroke Width and check the result after pressing \*_OK_.
+- In a similar way, you can edit the symbology of the first Shapefile Layer we added and change the **Fill Color**, **Stroke Color** and **Stroke Width**.
+- Do you still see the roads in New-York? Try to adjust the **Opacity** value (default is 1) to a lower value for this Shapefile Layer. Can you see all your layers now?
+
+```{exercise-end}
+
+```
+
+```{solution} add-vector-layer
+:class: dropdown
+
+After adding the new Shapefile Layer and zooming over New-York, you should have the following map:
+![View several shapefile Layer on the same map](../assets/images/severalShapefileLayers.png)
+
+When you right click to edit the symbology, you should get the following pop-up menu:
+![Edit symbology](../assets/images/editSymbology.png)
+
+And when editing the symbology of the other Shapefile Layer:
+![Adjust Symbology](../assets/images/adjustSymbology.png)
+
+And updating the opacity (for instance to 0.5) of the US county Shapefile Layer, you can get a map that can look like to (depending on the colors and Stroke width you chose!):
+
+![final map for the intro exercise](../assets/images/finalIntroMap.png)
+```
+
+## Hiding and Reordering Layers
+
+### Hiding/Viewing Layers
+
+In the GIS Layers List/Browser Panel, you can select a layer and click the **eye** icon to hide or show the corresponding layer.
+![Hide a layer](../assets/images/hideLayer.png)
+
+### Reordering Layers
+
+The layers in your Layers List are displayed on the map according to their order in the list. The bottom layer is drawn first, while the top layer is drawn last. You can adjust their drawing order by rearranging the layers in the list.
+
+The order in which the layers have been loaded into the map is probably not logical at this stage. It’s possible that the road layer or/and the country layer are completely hidden because other layers are on top of it.
+
+For example, this layer order:
+![Wrong layer order](../assets/images/wrongLayerOrder.png)
```

Comment:
I think we should only show layers in this screenshot, not sources. It's a bit confusing!